src/WarcraftLegacies.Source/Quests/Stormwind/QuestHonorHold.cs

```diff
@@ -34,7 +34,7 @@ public sealed class QuestHonorHold : QuestData
 
     /// <inheritdoc/>
     protected override string RewardDescription =>
-      "Control of all units at Honor Hold and the ability to train Siege Towers";
+      $"Control of all units at Honor Hold and {GetObjectName(Constants.UNIT_O06K_SIEGE_TOWER_STORMWIND)} gain the Artillery Bombardment ability."; 
```

Comment:
Use `GetObjectName` for ability names as well, works the same way. Also no full stop at the end needed here.